Makefile

```diff
@@ -2,37 +2,37 @@ testjs: ## Clean and Make js tests
 	yarn test
 
 testpy: ## Clean and Make py tests
-	python3.7 -m pytest -v jupyterfs/tests --cov=jupyterfs --cov-branch --junitxml=python --cov-report=xml
+	python3.7 -m pytest -v jupyterfs/tests --cov=jupyterfs --cov-branch --junitxml=python_junit.xml --cov-report=xml
 
 test: ## run all tests
-	testpy
-	testjs
+	make testpy
+	make testjs
 
 lintjs: ## run linter
 	yarn lint
 
 lintpy: ## run linter
-	flake8 jupyterfs setup.py
+	python3.7 -m flake8 jupyterfs setup.py
```

Comment:
Is the defensive use of `python3.7 -m` a Windows thing, or is it just to ensure the specific python version?

Reply:
Not windows but I always have issues, for example my flake8 currently uses python2 in its wrapper shell script (I must have updated it most recently). With python3.8 now half my shell scripts are gonna map to that